.github/workflows/integration-tests.yaml

```diff
@@ -96,3 +96,11 @@ jobs:
           python3 -m pip install -e sdk/python; pytest sdk/python/test --log-cli-level=info --namespace=default
         env:
           GANG_SCHEDULER_NAME: ${{ matrix.gang-scheduler-name }}
+
+      - name: Collect volcano logs
+        if: ${{ failure() &&  env.GANG_SCHEDULER_NAME == 'volcano' }}
+        run: |
+          kubectl logs $(kubectl get po -n volcano-system | grep volcano-scheduler |cut -d" " -f1) -n volcano-system > volcano-scheduler.log
+          kubectl logs $(kubectl get po -n volcano-system | grep volcano-admission |cut -d" " -f1 | grep -v init) -n volcano-system > volcano-admission.log
+          kubectl logs $(kubectl get po -n volcano-system | grep volcano-controllers |cut -d" " -f1) -n volcano-system > volcano-controllers.log
```

Comment:
Should we show logs instead of writing logs to files?

Reply:
I think the same.